src/controllers/checkoutController.js

```diff
@@ -0,0 +1,112 @@
+import * as dotenv from 'dotenv';
+import Stripe from 'stripe';
+
+import db from '../database/models/index';
+import { cart_retriver } from '../helpers/cart_retriever';
+
+dotenv.config();
+const { User, Order, OrderItem, Product, Cart } = db;
+const stripe = new Stripe(process.env.STRIPE_API_KEY);
```

Comment:
why can't create a help function for Stripe then it will be used in any form
and it must have the same parameters like ` mode: 'payment',
      customer: customer.id, item and payment method by default make it card`
      in return provide `session`